### PR TITLE
Bug 1942548: i18n translation of create route form

### PR DESCRIPTION
**Pull request by sg00dwin**
Bug https://bugzilla.redhat.com/show_bug.cgi?id=1942548

### CODE REVIEW

File: frontend/public/components/routes/create-route.tsx [hunk 1]
```diff
@@ -335,7 +339,7 @@ export class CreateRoute extends React.Component<{}, CreateRouteState> {
                 isInline
               >
                 <MinusCircleIcon className="co-icon-space-r" />
-                Remove Alternate Service
+                {t('public~Remove Alternate Service')}
```
Comment (spadgett) [inline frontend/public/components/routes/create-route.tsx:342]:
```suggestion
                {t('public~Remove alternate Service')}
```

File: frontend/public/components/routes/create-route.tsx [hunk 2]
```diff
@@ -387,11 +391,11 @@ export class CreateRoute extends React.Component<{}, CreateRouteState> {
                 required
               />
               <div className="help-block" id="name-help">
-                <p>A unique name for the route within the project.</p>
+                <p>{t('public~A unique name for the route within the project.')}</p>
```
Comment (spadgett) [inline frontend/public/components/routes/create-route.tsx:394]:
```suggestion
                <p>{t('public~A unique name for the Route within the project.')}</p>
```

File: frontend/public/components/routes/create-route.tsx [hunk 3]
```diff
@@ -403,11 +407,15 @@ export class CreateRoute extends React.Component<{}, CreateRouteState> {
                 aria-describedby="hostname-help"
               />
               <div className="help-block" id="hostname-help">
-                <p>Public hostname for the route. If not specified, a hostname is generated.</p>
+                <p>
+                  {t(
+                    'public~Public hostname for the route. If not specified, a hostname is generated.',
```
Comment (spadgett) [inline frontend/public/components/routes/create-route.tsx:412]:
```suggestion
                    'public~Public hostname for the Route. If not specified, a hostname is generated.',
```

File: frontend/public/components/routes/create-route.tsx [hunk 4]
```diff
@@ -433,7 +441,7 @@ export class CreateRoute extends React.Component<{}, CreateRouteState> {
                   variant="info"
                   title="No services"
                 >
-                  There are no services in your project to expose with a route.
+                  {t('public~There are no services in your project to expose with a route.')}
```
Comment (spadgett) [inline frontend/public/components/routes/create-route.tsx:444]:
```suggestion
                  {t('public~There are no Services in your project to expose with a Route.')}
```

File: frontend/public/components/routes/create-route.tsx [hunk 5]
```diff
@@ -483,14 +492,14 @@ export class CreateRoute extends React.Component<{}, CreateRouteState> {
                   isInline
                 >
                   <PlusCircleIcon className="co-icon-space-r" />
-                  Add Alternate Service
+                  {t('public~Add Alternate Service')}
```
Comment (spadgett) [inline frontend/public/components/routes/create-route.tsx:495]:
```suggestion
                  {t('public~Add alternate Service')}
```

File: frontend/public/components/routes/create-route.tsx [hunk 6]
```diff
@@ -483,14 +492,14 @@ export class CreateRoute extends React.Component<{}, CreateRouteState> {
...
                 </Button>
               )}
             <div className="form-group co-create-route__target-port">
               <label className="co-required" htmlFor="target-port">
-                Target Port
+                {t('public~Target port')}
               </label>
-              {_.isEmpty(portOptions) && <p>Select a service above</p>}
+              {_.isEmpty(portOptions) && <p>{t('public~Select a service above')}</p>}
```
Comment (spadgett) [inline frontend/public/components/routes/create-route.tsx:502]:
```suggestion
              {_.isEmpty(portOptions) && <p>{t('public~Select a Service above')}</p>}
```

File: frontend/public/components/routes/create-route.tsx [hunk 7]
```diff
@@ -517,20 +526,21 @@ export class CreateRoute extends React.Component<{}, CreateRouteState> {
                     name="secure"
                     aria-describedby="secure-help"
                   />
-                  Secure route
+                  {t('public~Secure route')}
```
Comment (spadgett) [inline frontend/public/components/routes/create-route.tsx:529]:
```suggestion
                  {t('public~Secure Route')}
```

File: frontend/public/components/routes/create-route.tsx [hunk 8]
```diff
@@ -657,7 +677,7 @@ export const AlternateServicesGroup: React.FC<AlternateServiceEntryGroupProps> =
   return (
     <>
       <div className="form-group">
-        <label htmlFor={`${index}-alt-service`}>Alternate Service Target</label>
+        <label htmlFor={`${index}-alt-service`}>{t('public~Alternate Service Target')}</label>
```
Comment (spadgett) [inline frontend/public/components/routes/create-route.tsx:680]:
```suggestion
        <label htmlFor={`${index}-alt-service`}>{t('public~Alternate Service target')}</label>
```

File: frontend/public/components/routes/create-route.tsx [hunk 9]
```diff
@@ -667,11 +687,11 @@ export const AlternateServicesGroup: React.FC<AlternateServiceEntryGroupProps> =
           describedBy={`${index}-alt-service-help`}
         />
         <div className="help-block" id={`${index}-alt-service-help`}>
-          <p>Alternate service to route to.</p>
+          <p>{t('public~Alternate service to route to.')}</p>
```
Comment (spadgett) [inline frontend/public/components/routes/create-route.tsx:690]:
```suggestion
          <p>{t('public~Alternate Service to route to.')}</p>
```

File: frontend/public/components/routes/create-route.tsx [hunk 10]
```diff
@@ -667,11 +687,11 @@ export const AlternateServicesGroup: React.FC<AlternateServiceEntryGroupProps> =
...
         </div>
       </div>
       <div className="form-group">
-        <label htmlFor={`${index}-weight`}>Alternate Service Weight</label>
+        <label htmlFor={`${index}-weight`}>{t('public~Alternate Service Weight')}</label>
```
Comment (spadgett) [inline frontend/public/components/routes/create-route.tsx:694]:
```suggestion
        <label htmlFor={`${index}-weight`}>{t('public~Alternate Service weight')}</label>
```

**Review by spadgett — COMMENTED** [review]
Looks good. We should update the case to conform with PatternFly standards.

**Comment by sg00dwin** [comment]
Case change comments have been addressed

File: frontend/public/components/routes/create-route.tsx [hunk 11]
```diff
@@ -600,7 +617,7 @@ export class CreateRoute extends React.Component<{}, CreateRouteState> {
                             onChange={this.onDestinationCACertificateChange}
                             inputFileData={this.state.destinationCACertificate}
                             id="destination-ca-certificate"
-                            label="Destination CA Certificate"
+                            label={t('public~Destination CA Certificate')}
```
Comment (spadgett) [inline frontend/public/components/routes/create-route.tsx:620]:
```suggestion
                            label={t('public~Destination CA certificate')}
```

File: frontend/public/components/routes/create-route.tsx [hunk 12]
```diff
@@ -555,43 +565,50 @@ export class CreateRoute extends React.Component<{}, CreateRouteState> {
...
                         />
                       </div>
                       <div className="form-group co-create-route__caCertificate">
                         <DroppableFileInput
                           onChange={this.onCaCertificateChange}
                           inputFileData={this.state.caCertificate}
                           id="ca-certificate"
-                          label="CA Certificate"
-                          inputFieldHelpText="The PEM format CA certificate chain. Upload file by dragging &amp; dropping, selecting it, or pasting from the clipboard."
+                          label={t('public~CA Certificate')}
```
Comment (spadgett) [inline frontend/public/components/routes/create-route.tsx:608]:
```suggestion
                          label={t('public~CA certificate')}
```

File: frontend/public/components/routes/create-route.tsx [hunk 13]
```diff
@@ -555,43 +565,50 @@ export class CreateRoute extends React.Component<{}, CreateRouteState> {
...
                         />
                       </div>
                       <div className="form-group co-create-route__private-key">
                         <DroppableFileInput
                           onChange={this.onPrivateKeyChange}
                           inputFileData={this.state.key}
                           id="private-key"
-                          label="Private Key"
-                          inputFieldHelpText="The PEM format key. Upload file by dragging &amp; dropping, selecting it, or pasting from the clipboard."
+                          label={t('public~Private Key')}
```
Comment (spadgett) [inline frontend/public/components/routes/create-route.tsx:597]:
```suggestion
                          label={t('public~Private key')}
```

Comment (sg00dwin) [reply frontend/public/components/routes/create-route.tsx]: updated

File: frontend/public/components/routes/create-route.tsx (hunk 12 above)
Comment (sg00dwin) [reply frontend/public/components/routes/create-route.tsx]: updated

File: frontend/public/components/routes/create-route.tsx (hunk 11 above)
Comment (sg00dwin) [reply frontend/public/components/routes/create-route.tsx]: updated

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**